docs/samples/k8s/hello-world/solution2.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: solution.symphony/v1
+kind: Solution
+metadata: 
+  name: prometheus-v8
+spec:  
+  metadata:
+    deployment.replicas: "#1"
+    service.ports: "[{\"name\":\"port9090\",\"port\": 9090}]"
+    service.type: "LoadBalancer"
+  version: v8
+  rootResource: prometheus:v8
```

Comment:
Is this a current input? I think you want rootResource as prometheus and version as v8.